test/torch_np/test_reductions.py

```diff
@@ -259,7 +269,42 @@ def test_sum_where(self):
         )
 
 
-class _GenericReductionsTestMixin:
+parametrize_axis = parametrize(
+    "axis", [0, 1, 2, -1, -2, (0, 1), (1, 0), (0, 1, 2), (1, -1, 0)]
+)
+parametrize_func = parametrize(
+    "func",
+    [
+        np.any,
+        np.all,
+        np.argmin,
+        np.argmax,
+        np.min,
+        np.max,
+        np.mean,
+        np.sum,
+        np.prod,
+        np.std,
+        np.var,
+        np.count_nonzero,
+    ],
+)
+
+fails_axes_tuples = {
+    np.any,
+    np.all,
+    np.argmin,
+    np.argmax,
+    np.prod,
+}
+
+fails_out_arg = {
+    np.count_nonzero,
+}
```

Comment:
this one we could probably fix?

Reply:
If we want to do a superset of numpy here, which does not have the out= arg :-) 

https://numpy.org/doc/stable/reference/generated/numpy.count_nonzero.html

Reply:
oh, it doesn't. So this is expected then? It should probably be a skip then (no need to act on this)